central/image/service/http_handler.go

```diff
@@ -187,19 +194,24 @@ func (h sbomHttpHandler) getScannerV4SBOMIntegration() (scannerTypes.SBOMer, err
 			}
 		}
 	}
-	return nil, errors.New("Scanner v4 integration not found")
+	return nil, errors.New("Scanner V4 integration not found")
 }
 
-// scannedByScannerv4 checks if image is scanned by scanner v4
+// scannedByScannerv4 checks if image is scanned by Scanner V4.
 func (h sbomHttpHandler) scannedByScannerv4(img *storage.Image) bool {
 	return img.GetScan().GetDataSource().GetId() == iiStore.DefaultScannerV4Integration.GetId()
 }
 
-// saveImage saves the image to the scanner database
+// saveImage saves the image to Central's database.
```

Comment:
Seems like this is only true if there is a SHA. When would be a case when we don't have a SHA and why would that stop us from saving it?